src/pages/for-companies/ForCompanies.tsx

```diff
@@ -31,17 +32,28 @@ const ForCompanies: React.FC<ForCompaniesProps> = () => {
         <li>14:15 - 18:00 - Standområdet er åpent</li>
         <li>18:00 - 20:00 - Nedrigging av lokalet</li>
       </ul>
-      <h5>Praktisk informasjon</h5>
+      <PageTitle>Praktisk informasjon</PageTitle>
```

Comment:
Burde kanskje lagd noen egne komponenter for subtitles, `PageTitle` er jo egentlig kun for tittelen på siden. @JonasTriki

Reply:
Ja, godt poeng. PageTitle er hovedsakelig for tittelen på siden.

Reply:
Får vel fikse det i en annen PR.